index.js

```diff
@@ -521,6 +544,11 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this.valueOf() <= other.valueOf();
+  }
+
```

Comment:
Additionally, Sanctuary provides the `is` function and `is(Boolean, false) === true`. This is confusing if `equals(new Boolean(false), false) === false`.